src/cmd/build.js

```diff
@@ -116,6 +119,13 @@ export class FileFilter {
         return false;
       }
     }
+    for (const dirPath of this.dirToIgnore) {
+      if (dirPath === path){
```

Comment:
these could paths could be files or directories so I think you should name it something more like `this.filePathsToIgnore`.